torch/_inductor/ir.py

```diff
@@ -3141,12 +3153,33 @@ class MultiOutputLayout(IRNode):
 
 
 class MultiOutput(ExternKernel):
+    def gen_list_tuple_access(self, basename, index):
+        # cpp wrapper code needs to use std::get<> to access a tuple
+        if index.startswith("["):
```

Comment:
this code is pretty awkward, maybe we should store index as an integer list instead of str?